qlib/Mapper.qm

```diff
@@ -1044,6 +1067,7 @@ Mapper mapv(DataMap);
 
             # add value to row list
             if (m.ostruct) {
+                # NOTE: ostruct is not supported in list mode # FIXME how we ensure it?
```

Comment:
same as the comment with `struct` - this works; tests have already been merged to 0.8.12 - this comment needs to be removed